applications/sim_showers_for_trigger_rates.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application simulates showers to be used in trigger rate calculations.
+    Arrays with one (1MST) or four telescopes (4LST) can be used, in case of \
```

Comment:
Only 1 or 4 telescopes, or also 2 or 5?

Reply:
We don't use anything other than 1 or 4 telescope arrays for trigger rate estimations. Actually we use only 1SST, 1MST and 4LST.
If this change in the future, we can build more layouts and change the documentation.